tests/e2e/hardhat/hardhat.config.js

```diff
@@ -0,0 +1,20 @@
+require("@nomiclabs/hardhat-ethers");
+const fs = require("fs");
+
+task('node', 'Starts a Hardhat node', async (args, hre, runSuper) => {
+  const accounts = config.networks.hardhat.accounts;
```

Comment:
This is to save the default hardhat network accounts mnemonic to a shared volume where it can be read to create a wallet for contract interactions. 

Alternatively we could have an env file that specifies the mnemonic that gets read anywhere we need accounts to use.

Reply:
Awesome–could we call this first part out in an inline comment for posterity?